core/src/main/java/com/netease/arctic/utils/TablePropertyUtil.java

```diff
@@ -133,4 +134,19 @@ public static long getTableWatermark(Map<String, String> properties) {
       return Long.parseLong(watermarkValue);
     }
   }
+
+  /**
+   * Check if an iceberg table is a table store of a mixed-iceberg table.
+   *
+   * @param properties table properties of iceberg
+   * @return true if this it a table store.
+   */
+  public static boolean isMixedTableStore(Map<String, String> properties) {
```

Comment:
The Mixed Hive Format also has the same problem, and the unkeyed Mixed Format table should also be considered.

Reply:
The table store of mixed-hive will not be loaded by iceberg catalog